validationform.js

```diff
@@ -72,9 +72,11 @@ function validationform() {
 function check(e) {
   e.preventDefault();
   validationform();
-  /* if (validationform()) {
+  if (validationform()) {
     // eslint-disable-next-line no-undef
-    /* saveinfomation(); */
+    saveinfomation();
+    // eslint-disable-next-line no-alert
+    alert('submit sucessfull');
```

Comment:
- [ ] kindly solve linters instead of disabling them. Remove all your alert messages as these should not be present in your code for best practice.

Reply:
ok i disable the linters and delete all the alerts in all javascript